src/components/Nav/Nav.tsx

```diff
@@ -75,6 +84,12 @@ export function NavbarScreen() {
               </Fragment>
             )}
           </div>
+          {showProfileMenu && (
+            <div className="absolute left-0 right-0 top-full z-50 flex flex-col items-center justify-center gap-4 border bg-white p-4">
```

Comment:
You copied the mobile menu styles. Those styles are meant for mobile layouts. It's not common for menus like that to take up the full width on desktop layouts.

Consider just adding some buttons to the navbar.

Reply:
I made the menu smaller 
![image](https://github.com/hawks-atlanta/frontend-react/assets/131629166/8563c24b-15d6-4511-85f4-73c0a925a696)